llama_index/llms/litellm_utils.py

```diff
@@ -0,0 +1,275 @@
+import logging
+import os
+import re
+from typing import Any, Callable, Dict, List, Optional, Sequence, Type, Union
+
+import openai
+from litellm import completion
```

Comment:
This import should probably be moved to the function where it's used, and wrapped in a try/except to direct the user to installation instructions